### PR TITLE
Add the Adviser's team to an Interaction activity feed card

### PR DESCRIPTION
**Pull request by paulgain**
At present teams can ONLY be shown within the Interaction activity cards.

team

https://trello.com/c/jfrFN7X8/1301-add-team-to-interactions-activity-feed-card

### CODE REVIEW

File: src/activity-feed/activity-feed-cards/card/CardUtils.jsx [hunk 1]
```diff
@@ -6,14 +6,16 @@ import {
 import { Link } from 'govuk-react'
 import React from 'react'
 
-const createEmailAddressMarkup = ({ id, name, emailAddress }) => {
+const createEmailAddressMarkup = ({ id, name, emailAddress, teamName }) => {
   if (!name || !emailAddress) {
     return null
   }
 
+  const formattedEmailAddress = teamName ? `${emailAddress},` : emailAddress
```
Comment (rafenden) [inline src/activity-feed/activity-feed-cards/card/CardUtils.jsx]: Maybe having an array of name "parts" could simplify this function?

Example:
```
const nameParts = [
  name,
  <Link href={`mailto:${emailAddress}`}>{formattedEmailAddress}</Link>,
  teamName,
]
```

And then filter the empty ones and join the rest, example:
```
return nameParts.filter(n => !n).join(', ')
```

File: src/activity-feed/activity-feed-cards/card/CardUtils.jsx [hunk 2]
```diff
@@ -6,14 +6,16 @@ import {
 import { Link } from 'govuk-react'
 import React from 'react'
 
-const createEmailAddressMarkup = ({ id, name, emailAddress }) => {
+const createEmailAddressMarkup = ({ id, name, emailAddress, teamName }) => {
```
Comment (rafenden) [inline src/activity-feed/activity-feed-cards/card/CardUtils.jsx]: I would move this function into a separate component, example:

```
<CardPerson person={person} />
```
where `person` is an object:

```
person = {
  name: 'John',
  email: 'john@doe.bla',
  ...
}
```

File: src/activity-feed/__snapshots__/ActivityFeedCard.test.jsx.snap [hunk 3]
```diff
@@ -1827,8 +1845,10 @@ exports[`ActivityFeedCard when there is an interaction should render interaction
                     href="mailto:mariobros@dit.com"
                     muted={false}
                   >
-                    mariobros@dit.com
+                    mariobros@dit.com,
```
Comment (rafenden) [inline src/activity-feed/__snapshots__/ActivityFeedCard.test.jsx.snap]: I think the comma should be outside of the anchor.

File: src/activity-feed/activity-feed-cards/card/CardUtils.jsx (hunk 1 above)
Comment (paulgain) [reply src/activity-feed/activity-feed-cards/card/CardUtils.jsx]: This doesn't add any value, all it does is add a few more lines of code to achieve the same thing.

**Pull request closed without merging**